.circleci/config.yml

```diff
@@ -1,52 +1,219 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@volatile
```

Comment:
I would like to keep using the orb if possible as it makes maintaining consitency between the plugins easier.
Would you be open on using the orb if support for go 1.18 is added soon?

cc @mickmister

Reply:
Hi @hanzei; yes [now that the orb is updated](https://github.com/mattermost/circleci-orbs/pull/32) it makes sense, just had to do this in the meantime.

Reply:
@JulienTant Awesome thank you!